### DISCUSSION TITLE
Mini graph

### DISCUSSION THREAD

**Discussion by eyalco555** [GuyKh/iec-custom-component#158] [Q&A]
Hi Guy, How can I apply the data to a mini graph card? 
All the sensors give a single-valued value. (per hour, yesterday, today, this month, last month)
 I want to see the "energy usege" graph in a mini graph card. Is this possible?

**Comment by GuyKh** [comment]
If you want to see historical graph you can see it in the Energy tab.
as for showing it as a card, i don't know

**Reply by GuyKh** [reply]
Perhaps you can use Statistics Card
https://www.home-assistant.io/dashboards/statistics-graph/

**Reply by GuyKh** [reply]
For example:
```
chart_type: bar
period: day
type: statistics-graph
entities:
  - iec:iec_meter_123456_energy_consumption
stat_types:
  - state
title: Energy Consumption in the last 30 days
hide_legend: true
```

![image](https://github.com/user-attachments/assets/a48d34b8-75db-43cb-8969-1f2a66a4c59e)
